src/chapters/azure-intro/networking.rst

```diff
@@ -23,6 +31,8 @@ Examples of valid IP addresses:
 
    For the purposes of this class we will only work with IPv4 addresses. It should be noted that a new format for IP addresses (IPv6) is also used, but goes beyond the scope of this class. Everything you learn about IPv4 addresses in this class can be applied directly to IPv6 addresses with the exception of the numbering pattern.
 
+You have already worked with one special IP address the **loopback** IP Address. The loopback IP Address is represented as ``127.0.0.1`` and when requests are made to this address the request is sent back to the machine that made the request. We have done this throughout the class when running our C#.NET web applications, and to access the web app we would make a request to 127.0.0.1 in our browser.
```

Comment:
minor change for clarity:

> ...when requests are made to this address the request is *looped back* within the machine that issued it.

could also throw in a tip to help them understand the "why" 

```rst
.. admonition:: fun fact

  The Loopback Interface was designed specifically for developers to simulate networking from within a single host machine. The IP address ``127.0.0.1`` or *home IP* is mapped to the aptly named host name ``localhost`` which you have undoubtedly used many times!
```